frontend/src/components/Listing/ListingCard.tsx

```diff
@@ -6,54 +6,57 @@ import CardMedia from '@mui/material/CardMedia';
 import Typography from '@mui/material/Typography';
 import { Button, CardActionArea, CardActions } from '@mui/material';
 import { capitalize } from '@mui/material/utils';
-import AddShoppingCartIcon from '@mui/icons-material/AddShoppingCart';
 import AddEditIcon from '@mui/icons-material/Edit';
 import 'src/styles/ListingCard.css'
-import {useParams} from "react-router-dom";
+import { useParams } from "react-router-dom";
 import DialogBox from "../Dialog/DialogBox";
 import SellerListingForm from "../Seller/SellerListingForm";
-import {useContext} from "react";
-import {UserLoginContext} from "../../context/userLoginContext";
+import { useContext } from "react";
+import { UserLoginContext } from "../../context/userLoginContext";
 import { useLocation } from 'react-router-dom'
+import 'src/styles/ListingCard.css'
+import AddToCartButton from './AddToCartButton';
+
 export interface IListingCardProps {
-    cardName: string,
-    title: string,
-    author: string,
-    price: string,
-    image: string,
-    alt: string,
-    listingId : string,
-    isbn: string,
-    publisher: string,
-    description: string,
-    inventory: string,
-    releaseDate: string,
```

Comment:
List linting issue, please look into for future tickets

Reply:
Updated the linting to be 2spaced